fetch.bs

```diff
@@ -81,6 +83,12 @@ url:https://tools.ietf.org/html/rfc7234#section-1.2.1;text:delta-seconds;type:df
     },
     "OCSP": {
         "aliasOf": "RFC2560"
+    },
+    "HEADER-STRUCTURE": {
+        "authors": ["Mark Nottingham","Paul-Henning Kamp"],
+        "href": "https://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html",
```

Comment:
I suppose we should use the tools.ietf.org version, unless we want to update all?

Reply:
Shifted to https://tools.ietf.org/html/draft-ietf-httpbis-header-structure.